### DISCUSSION TITLE
CommandItem color gray instead of black

### DISCUSSION THREAD

**Discussion by MickaelMoon** [shadcn-ui/ui#4203] [General]
Hello,
I was doing update in my shadcn/ui component, and i saw a update for the command component, but only 1 line as been changed and these swaped the color and also broke the CommandItem function onSelect isn't working.
<img width="1512" alt="Capture d’écran 2024-07-07 à 12 21 19" src="https://github.com/shadcn-ui/ui/assets/60145073/1d00ca37-36dc-458c-ab28-966c11e64dd6">
<img width="1512" alt="Capture d’écran 2024-07-07 à 12 21 53" src="https://github.com/shadcn-ui/ui/assets/60145073/f5f45dcc-f19d-47db-a2a0-12f310676201">

**Comment by abhayclasher** [comment]
v1 of cmdk broke the dom structure and onSelect bubbling. just downgrade cmdk to 0.2.0 in ur package.json or rerun npx shadcn-ui@latest add command to get the updated v1 compatible component code.
